Flask_webapp/app.py

```diff
@@ -10,28 +10,41 @@
 
 app = Flask(__name__)
 
+onlyfiles = preload.list_models('/Flask_webapp/models')
+string = ""
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
     # relative path of models folder
     onlyfiles = preload.list_models('/Flask_webapp/models')
     # send ML models to index.html page, no value is taken from html pages
-    return render_template('index.html', modelfiles=onlyfiles)
+    return render_template('index.html', modelfiles=onlyfiles, string=string)
 
 
 @app.route('/predict', methods=['POST', 'GET'])
 def pred():
+    # if ticker not in request.form:
     ticker = request.form.get('ticker')
     select = request.form.get('comp_select')
+    try:
+        dataset = dataGenerator.data_fetch(ticker)
+    # except ImportError:
+    #     return "Please add a ticker!!"
+    except Exception:
+        # return "Invalid ticker!!"
+        return render_template('index.html', modelfiles=onlyfiles, string="ENTER A VALID TICKER!")
```

Comment:
nitpick (and doesn't matter too much here), but it's good practice to have friendlier error messages. all caps with an exclamation point is a bit aggressive.

A better message would be polite and also informative: e.g. "Could not find data for {ticker}"